interpreter/dotnet/instance.go

```diff
@@ -165,7 +165,8 @@ func (i *dotnetInstance) appendStubFrame(frames *libpf.Frames, codeType uint) {
 
 // addRange inserts a known memory mapping along with the needed data of it to ebpf maps
 func (i *dotnetInstance) addRange(ebpf interpreter.EbpfHandler, pid libpf.PID,
-	lowAddress, highAddress, mapBase libpf.Address, stubTypeOrHdrMap uint64) {
+	lowAddress, highAddress, mapBase libpf.Address, stubTypeOrHdrMap uint64,
+) {
```

Comment:
This PR has a ton of these unrelated stylistic changes. This makes rebasing other PRs really painful.
Please stop changing code unnecessarily.

Reply:
Sorry about that. Would it be fine for you if I create a specific PR to format all the Go code? Something like what we did in OpAMP-go and all other OpenTelemetry Go projects: https://github.com/open-telemetry/opamp-go/pull/356

Most IDEs or editors configurations auto-format on save.